howto/install-appliance/google-cloud.md

```diff
@@ -0,0 +1,107 @@
+The Anbox Cloud Appliance is not yet available from the Google Marketplace. However, you can install the Anbox Cloud Appliance snap on a Google Cloud instance by using this guide.
+
+The entire deployment process will take 20-30 minutes, depending on the selected hardware and the network conditions.
+
+## Prerequisites
+
+Before starting the procedure,
+* Check the hardware requirements for the Anbox Cloud Appliance [here](https://discourse.ubuntu.com/t/17734).
+* Make sure that you have a Google Cloud account and a project on Google Cloud to create the virtual machine.
+* If you wish to use your own Ubuntu Pro subscription, ensure you have the Ubuntu Pro token for your Ubuntu Pro subscription. If you wish to use the Ubuntu Pro subscription offered by Google Cloud along with the virtual machine, skip this prerequisite.
+
+## Virtual machine setup
+
+### 1. Create a virtual machine
+Log in to [Google Cloud](https://console.cloud.google.com) and select the project for this deployment. Select **CREATE INSTANCE** in the **VM instances** page.
+
+![VM instances on Google Cloud|690x440](https://assets.ubuntu.com/v1/d44bd0dd-VM-instance.png)
+
+Select **New VM instance** from the options on the sidebar.
+
+![VM instance options on Google Cloud|690x440](https://assets.ubuntu.com/v1/cbc94514-Instance-options.png)
+
+### 2. Configure basic settings
+
+Configure the following basic settings for the virtual machine. 
+
+* **Name** - Name of the virtual machine instance
+* **Labels** - Organisational labels to keep track of your resources on Google Cloud
+* **Region** and **Zone** - The geographic location where your resources are run and your data is stored
+* **Machine configuration**, **Series**, **Machine type** and **Display device** - Select a machine configuration that matches the hardware requirements. If your requirement includes GPUs, select the GPU-optimised machine configuration. For example, an NVIDIA L4 GPU. You can select the recommended preset machine type or define a custom type with additional cores and memory for an added cost.
+
+Most of the configuration depends on your deployment and its location. For the settings that are not mentioned in this guide, you can choose to proceed with the default options or see [Google documentation](https://cloud.google.com/compute/docs/instances/create-start-instance) to customise your virtual machine on Google Cloud.
```

Comment:
I'm not sure, but:
```suggestion
Most of the configuration depends on your deployment and its location. For the settings that are not mentioned in this guide, you can choose to proceed with the default options or see [Google's documentation](https://cloud.google.com/compute/docs/instances/create-start-instance) to customise your virtual machine on Google Cloud.
```
That feels more natural to me, but I may be wrong. What do you think?